frontend/src/scenes/annotations/index.tsx

```diff
@@ -208,76 +227,85 @@ function CreateAnnotationModal(props: CreateAnnotationModalProps): JSX.Element {
         }
     }, [props.annotation])
 
-    const _onSubmit = (input: string, date: dayjs.Dayjs): void => {
-        props.onSubmit(input, date)
+    const _onSubmit = (input: string, date: dayjs.Dayjs, annotationScope: AnnotationScope): void => {
+        props.onSubmit(input, date, annotationScope)
         // Reset input
         setTextInput('')
         setDate(dayjs())
         setScope(AnnotationScope.Project)
     }
 
     return (
-        <Modal
-            footer={[
-                <Button key="create-annotation-cancel" onClick={(): void => props.onCancel()}>
-                    Cancel
-                </Button>,
-                <Button
-                    type="primary"
-                    key="create-annotation-submit"
-                    data-attr="create-annotation-submit"
-                    onClick={(): void => {
-                        _onSubmit(textInput, selectedDate)
-                    }}
-                >
-                    {modalMode === ModalMode.CREATE ? 'Submit' : 'Update'}
-                </Button>,
-            ]}
-            closable={false}
-            visible={props.visible}
-            onCancel={props.onCancel}
+        <LemonModal
+            isOpen={props.visible}
+            footer={
+                <div className="flex flex-row justify-end gap-2">
```

Comment:
_I think_ this is basically what the footer adds anyways. You can likely remove the enclosing div and it will look identical / how the design system says it should look